modules/sdk/gradle-plugins-maven-plugin-builder/src/main/java/com/liferay/gradle/plugins/maven/plugin/builder/BuildPluginDescriptorTask.java

```diff
@@ -387,6 +427,24 @@ protected void buildPomFile(File pomFile, File sourceDir) throws Exception {
 		XMLUtil.appendElement(
 			document, projectElement, "packaging", "maven-plugin");
 
+		Element repositoriesElement = document.createElement("repositories");
```

Comment:
Can we just avoid adding the `<repositories>` element if there are no `pomRepositories` (that is, if the map is empty)? Just to keep things clean :blush: